### ISSUE TITLE
[Android] Using 4.4x release causes auto-update to regular, broken release

### ISSUE THREAD

**Issue by superuser-does** [veloce/lichobile#847]
Bug report

**Current behavior:**

Installing the 4.4 release registers lichess on the Play Store. This means it receives automatic updates.

Unfortunately, it is automatically updated to a non-functional version. My dad has this issue on his Android 5.1 phone (which seems to have lost support as well) and it took 20 minutes of workarounds; he would have given up on the app if I wasn't there (still not sure how he found the right release though, to his credit).

**Expected behavior:**

It would be best if there is no auto-update function at all.

**Steps to reproduce:**

1. Install the 4.4 APK
2. Have it auto-update
3. Boom, your app is now broken

**Environment information**

- phone hardware: _Blackview Alife P1 Pro_
- phone operating system version: _Android 5.1_
- version of lichess mobile: 6.0.1

**Other information:**

A workaround is to go in the Play Store on data, disable auto-updates for lichess and keep installing lichess updates from the releases page.

**Comment by veloce** [comment]
There are 2 apks uploaded to the store, one for 4.4 and one for > 5.0 android versions. It seems that for you the >5.0 apk is broken, so you install the 4.4 one. Is that correct? The only 4.4 apk is for version 6.0.1, so I assume the app works for you with that version and that apk.

Now, I'd like to know more:
- which app versions number does *not* work on your phone with the 5.0 apk 6.0.0?, 6.0.1?, both? 
- a description of the problem on those versions that don't work.

That would help. Thanks.

**Comment by superuser-does** [comment]
> which app versions number does not work on your phone with the 5.0 apk 6.0.0?, 6.0.1?, both?

It was 6.0.1 only, my bad, I edited the original post. We never tried 6.0.0. As it's working now my dad won't let me try it again, haha.

So from what I've gathered, if you install the 4.4 apk, the Play Store will download the alternate version anyway even if no updates are needed.

>a description of the problem on those versions that don't work.

The app loads and he sees the horse, but then instead showing the home page it goes to a blank screen and stays like that, without any points of interaction displayed. Doesn't crash, notably.
